inst/scripts/SEL3D.R

```diff
@@ -0,0 +1,224 @@
+#SEL3D.R
+#
+#' \code{getPdbIds} BLASTs the input cDNA sequence against PDB and returns an MT object
+#' containing the blast matches from PDB for homologous (>40% ID) sequences.
+#'
+#' Details.
+#' @section <title>: Additional explanation.
+#'
+#' @param targetSeq input sequence to be queried, must be cDNA of a length divisible by 3.
+#' @param startPos start position of the sequence, set to 1 by default, altered in recursive steps
+#' @param endPos end position of sequence, set to 0, altered to aa length in recursive steps
```

Comment:
startPos and endPos are parameters for your helper function getPdbIds and not for SEL3D. SEL3D and getPdbIds should both have their own function descriptions.